develop/index.md

```diff
@@ -6,29 +6,189 @@ hide:
+    - [**XCM Tools**](/develop/toolkit/interoperability/xcm-tools/){target=\_blank} - provides an overview of popular XCM tools
+    - [**Tutorials for Managing XCM Channels**](/tutorials/interoperability/xcm-channels/){target=\_blank} - guides for using [Polkadot.js Apps](https://polkadot.js.org/apps/#/explorer){target=\_blank} UI to establish cross-chain messaging channels 
 
-- **Cross-chain developers** - harness Polkadot's interoperability features to enable secure communication and asset transfers across blockchains. Leverage Cross-Consensus Messaging (XCM) to create innovative cross-chain workflows and applications
 
-    - **Where to start** - [Introduction to XCM](/develop/interoperability/intro-to-xcm/){target=\_blank}
+### Parachain Developers 
+
+Build, deploy, and maintain custom parachains with the Polkadot SDK.
+
+####  Learn
+
+<div class="grid cards" markdown>
+
+- :octicons-book-16:{ .lg .middle } [Parachains Overview](/polkadot-protocol/architecture/parachains/overview/){target=\_blank}
+
+- :octicons-book-16:{ .lg .middle } [Custom Parachains](/develop/parachains/get-started/build-custom-parachains/){target=\_blank}
+
+- :octicons-book-16:{ .lg .middle } [Introduction to the Polkadot SDK](/develop/parachains/get-started/intro-polkadot-sdk/){target=\_blank}
+
+</div>
+
+#### Build
+
+<div class="grid cards" markdown>
+
+
+- :octicons-rocket-16:{.lg .middle} [Parachain Developer Zero to Hero Guides](tutorials/polkadot-sdk/parachains/zero-to-hero/){target=\_blank}
+
+- :octicons-rocket-16:{.lg .middle} [Install Polkadot SDK Dependencies](/develop/parachains/get-started/install-polkadot-sdk/){target=\_blank}
+
+</div>
+
+#### Tools
+
+<div class="grid cards" markdown>
+
+- :octicons-tools-16:{.lg .middle} [Parachain Template](https://github.com/paritytech/polkadot-sdk/tree/master/templates/parachain){target=\_blank}
+
+- :octicons-tools-16:{.lg .middle} [Open Zeppelin Polkadot Runtime Templates](https://github.com/OpenZeppelin/polkadot-runtime-templates/tree/main){target=\_blank}
+
+</div>
+
+#### Resources
+
+<div class="grid cards" markdown>
+
+- :octicons-code-square-16:{.lg .middle} [Polkadot SDK source code repository](https://github.com/paritytech/polkadot-sdk){target=\_blank} 
+    
+- :octicons-code-square-16:{.lg .middle} [FRAME source code repository](https://github.com/paritytech/polkadot-sdk/tree/master/substrate/frame){target=\_blank}
+
+</div>
+
+### Smart Contract Developers 
+
+Leverage smart contracts and execute custom logic over existing chains to streamline your development process.
+
+!!! tip
+    The Polkadot smart contract ecosystem is in active development. Please expect frequent changes. You can follow progress, or join the discussion, by visiting this [Contracts on AssetHub Roadmap](https://forum.polkadot.network/t/contracts-on-assethub-roadmap/9513/57){target=\_blank} Polkadot Network Forum post.
+
+
+#### Learn 
+
+<div class="grid cards" markdown>
```

Comment:
I like the idea of bringing in a few icons to break up the text. However, I'm not sure about this layout. I know this might not make sense, because whitespace can typically help with the ability to quickly scan a page, but this layout for me almost reduces that ability because now my eyes are frequently shifting left to right to try to catch everything.

Just curious, because if I am remembering correctly (which I might not be haha), I thought there was at one time a card for Learn, Build, etc. - may I ask what prompted you to change it? Did you play around with any other options?